lib/components/form/advanced-settings-panel.tsx

```diff
@@ -0,0 +1,274 @@
+  background: transparent;
+  border: none;
+`
+
+const HeaderContainer = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  height: 30px;
+`
+
+const Subheader = styled.h2`
+  ${invisibleCss}
+`
+
+const ReturnToTripPlanButton = styled.button`
+  align-items: center;
+  background-color: var(--main-base-color, ${blue[900]});
+  border: 0;
+  color: white;
+  display: flex;
+  font-weight: 700;
+  gap: 5px;
+  height: 51px;
+  justify-content: center;
+  margin-top: 2em;
+  width: 100%;
+
+  svg {
+    margin-bottom: 7px;
+  }
+`
+
+const DtSelectorContainer = styled.div`
+  margin: 2em 0;
+
+  .date-time-modal {
+    padding: 0;
+
+    .main-panel {
+      margin: 0;
+
+      button {
+        padding: 6px 0;
+      }
+
+      .date-time-selector {
+        margin: 15px 0;
+      }
+    }
+  }
+`
+
+const AdvancedSettingsPanel = ({
+  closeAdvancedSettings,
+  enabledModeButtons,
+  innerRef,
+  modeButtonOptions,
+  modeSettingDefinitions,
+  modeSettingValues,
+  setQueryParam
+}: {
+  closeAdvancedSettings: () => void
+  enabledModeButtons: string[]
+  innerRef: RefObject<HTMLDivElement>
+  modeButtonOptions: ModeButtonDefinition[]
+  modeSettingDefinitions: ModeSetting[]
+  modeSettingValues: ModeSettingValues
+  onPlanTripClick: () => void
+  setQueryParam: (evt: any) => void
+}): JSX.Element => {
+  const [closingBySave, setClosingBySave] = useState(false)
+  const [closingByX, setClosingByX] = useState(false)
+  const baseColor = getBaseColor()
+  const accentColor = baseColor || blue[900]
+
+  const intl = useIntl()
+  const closeButtonText = intl.formatMessage({
+    id: 'components.BatchSearchScreen.saveAndReturn'
+  })
+  const headerText = intl.formatMessage({
+    id: 'components.BatchSearchScreen.advancedHeader'
+  })
+
+  // @ts-expect-error Context not typed
+  const { ModeIcon } = useContext(ComponentContext)
+
+  const processSettings = (settings: ModeSetting[]) =>
+    settings.map(
+      pipe(
+        populateSettingWithIcon(ModeIcon),
+        populateSettingWithValue(modeSettingValues),
+        addCustomSettingLabels(intl)
+      )
+    )
+
+  const globalSettings = modeSettingDefinitions.filter((x) => !x.applicableMode)
+  const processedGlobalSettings = processSettings(globalSettings)
+
+  const globalSettingsComponents = processedGlobalSettings.map(
+    (setting: ModeSetting) => (
+      <ModeSettingRenderer
+        key={setting.key}
+        onChange={onSettingsUpdate(setQueryParam)}
+        setting={setting}
+      />
+    )
+  )
+
+  const processedModeSettings = processSettings(modeSettingDefinitions)
+  const processedModeButtons = modeButtonOptions.map(
+    pipe(
+      addModeButtonIcon(ModeIcon),
+      addSettingsToButton(processedModeSettings),
+      setModeButtonEnabled(enabledModeButtons)
+    )
+  )
+
+  return (
+    <PanelOverlay className="advanced-settings" ref={innerRef}>
+      <HeaderContainer>
+        <PageTitle title={headerText} />
```

Comment:
This doesn't seem to be working. It flashes advanced preferences and then goes back to the default. We can probably remove this feature?

Reply:
Was seeing this on Firefox but if it's working for you let's hope it's a fluke!

Reply:
@binh-dam-ibigroup Let me know if you notice this!

Reply:
I've removed the `PageTitle` component from Advanced Settings but I can look into this further if we feel like it's important to update the website title when this panel is open.